pages/about.html

```diff
@@ -4,9 +4,58 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
+  <link href="/styles/about.css" rel="stylesheet">
+  <title>About Me</title>
 </head>
 <body>
-  
+  <section class="container">
+    <div class="header">
+      <header>
+        <h1>About Me</h1>
+      </header>
+    </div>
+
+    <div class="navigation-links">
+      <nav>
+        <ul>
+          <li>
+            <a href="http://127.0.0.1:5500/pages/portfolio.html">Portfolio</a>
```

Comment:
These hrefs need to be updated to provide the _path_ to the resource (not the URL to access the resource). This is because the resource is in the same project as this file. If you're linking off your site to another website like Github, then you can provide the URL to the href attr